src/systems/bit-media-frames.js

```diff
@@ -264,16 +361,19 @@ export function mediaFramesSystem(world) {
     const frame = mediaFrames[i];
 
     const captured = world.nid2eid.get(MediaFrame.capturedNid[frame]) || 0;
-    const colliding = captured && isColliding(world, frame, captured);
-
-    if (captured && hasComponent(world, Owned, captured) && !hasComponent(world, Held, captured) && colliding) {
-      snapToFrame(world, frame, captured);
-      physicsSystem.updateBodyOptions(Rigidbody.bodyId[captured], { type: "kinematic" });
+    const isCapturedOwned = hasComponent(world, Owned, captured);
+    const isCapturedHeld = findChildWithComponent(world, Held, captured);
+    const colliding = captured && isColliding(world, physicsSystem, frame, captured);
+    const isFrameDeleting = findAncestorWithComponent(world, Deleting, frame);
+    const isFrameOwned = hasComponent(world, Owned, frame);
+
+    if (captured && isCapturedOwned && !isCapturedHeld && !isFrameDeleting && colliding) {
```

Comment:
Was slightly uneasy when I saw the addition of the `Deleting` component at the start of reviewing this PR. I see here we are using it to prevent scaling the object down as the media frame it was in gets shrunk during the delete animation. It definitely seems really odd to have to think about that delete animation in systems and this definitely wont be the last time we have to... Don't have a nicer solution off the top of my head but we should definitely brainstorm on this more.